main.go

```diff
@@ -158,6 +164,19 @@ func main() {
 	}
 }
 
+func validateTileMatrixSet(tms tms20.TileMatrixSet, tileMatrixIDs []tms20.TMID) error {
```

Comment:
mss hier nog een test op toevoegen? of op DeviationStats

Reply:
true beide hadden tests moeten hebben. en het gebeurde nu bij iedere polygon, ook onnodig.